### PR TITLE
2023-08-13 Release Notes

### PR DESCRIPTION
**Pull request by CocoWang-wql**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+
+### Release notes
+* Features
+  * [Azure Container Networking Interface (CNI) Overlay](https://learn.microsoft.com/azure/aks/azure-cni-overlay) now fully supports Windows Server 2019 and 2022.
+  
+* Behavioral changes
+  * Container Insights Logs Addon (e.g. Monitoring addon) property is migrated from `AddonProfiles` to `Cluster profile`.
+  * Azure monitor metrics addon image is reverted from 07-28-2023 release back to the [06-26-2023 release](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-06-26-2023)
+  * Through an annotation, the creation of a [Private Link Service](https://cloud-provider-azure.sigs.k8s.io/topics/pls-integration/) automates to the Azure LoadBalancer, requiring users to only create Private Endpoint connections for private connectivity.
```
Comment (chasewilson) [inline CHANGELOG.md:18]:
```suggestion
  * [Automate the creation](https://cloud-provider-azure.sigs.k8s.io/topics/pls-integration/) and connection of a [Private Link Service](https://learn.microsoft.com/azure/private-link/private-link-service-overview) to an Azure LoadBalancer, only requiring users to create Private Endpoint connections for private connectivity.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+* Component Updates
+  * AKS Image cleaner eraser image bumped to [v1.2.0](https://github.com/eraser-dev/eraser/releases/tag/v1.2.0)
```
Comment (qpetraroia) [inline CHANGELOG.md:21]: A couple sentences under component updates are missing a period, can you add them please?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-08-13
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with Azure AD Integration (legacy). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to AKS-managed Azure AD automatically starting from 1st Dec. 2023. We recommend to update your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
```
Comment (kaarthis) [inline CHANGELOG.md:9]: Having a public doc link will be helpful here.

File: CHANGELOG.md (hunk 1 above)
Comment (CocoWang-wql) [reply CHANGELOG.md:18]: @kaarthis  I confirmed with them, no public doc, GH.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/en-us/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/en-us/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes
+* Features
+  * [Azure Container Networking Interface (CNI) Overlay](https://learn.microsoft.com/azure/aks/azure-cni-overlay) now fully supports Windows Server 2019 and 2022.
+  
+* Behavioral changes
+  * Azure monitor metrics addon image is reverted from 07-28-2023 release back to the [06-26-2023 release](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-06-26-2023) because 07-28-2023 release contains an issue that configmap processing is broken for $ in regex fields.
```
Comment (wangyira) [inline CHANGELOG.md:16]: could you please add the link to 07-28-2023 release the first time that release is mentioned

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,29 @@
...
+  * [Automate the creation](https://cloud-provider-azure.sigs.k8s.io/topics/pls-integration/) and connection of a [Private Link Service](https://learn.microsoft.com/azure/private-link/private-link-service-overview) to an Azure LoadBalancer, only requiring users to create Private Endpoint connections for private connectivity.
```
Comment (wangyira) [inline CHANGELOG.md:17]: The doc only mentions auto approval not auto creation? Which is the one that's recently released/is the doc updated?

**Review by wangyira — CHANGES_REQUESTED** [review]

**Review by charleswool — APPROVED** [review]

**Review by wangyira — APPROVED** [review]

**Pull request merged**